avogadro/core/molecule.h

```diff
@@ -583,6 +592,9 @@ class AVOGADROCORE_EXPORT Molecule
   UnitCell* m_unitCell;
   Array<Residue> m_residues;
 
+  // This will be stored from the last space group operation
+  unsigned short m_hallNumber = 0;
```

Comment:
@cryos 
What do you think about initializing class members at their declaration like this?

Personally, I think it is a nice addition in C++11, since it allows for a default value for all constructors that can be overridden (if desired) by a constructor's member initializer list.